tests/cross_functional/system_test/test_nfs_export_in_out_cluster.py

```
@@ -0,0 +1,2081 @@
+
+        # Create a mapping of mount points to their corresponding pods
+        mount_to_pod_map = {
+            test_folder_for_pod: pod_name,  # Original mount -> Original pod
+            restored_test_folder: restored_pod_name,  # Restored mount -> Restored pod
+            cloned_test_folder: cloned_pod_name,  # Cloned mount -> Cloned pod
+            final_restored_test_folder: final_restored_pod_name,  # Final mount -> Final pod
+        }
+        for mount_point, pname in mount_to_pod_map.items():
+            try:
+                log.info(
+                    f"Testing bidirectional I/O for mount point: {mount_point} and pod: {pname}"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)